guide/case-studies/blixt-wallet.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: guide
+title: Blixt Wallet
+description: This was a collaboration between Blixt wallet and the Bitcoin Design Community to help new users establish lightning channels quicker with Blixt wallet.
+nav_order: 5
+parent: Case studies
+permalink: /guide/case-studies/blixt-wallet/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-preview.jpg
+---
+
+<!--
+
+Editor's notes
+
+Illustration sources: https://www.figma.com/file/eyQDCaMoQvnRVcHzSsgrzs/Blixt-UX-Audit?node-id=17%3A770&t=cCzUFD516C4F6rGD-1
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-header.jpg"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-header@2x.jpg"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-header-mobile.jpg"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-header-mobile@2x.jpg"
+   alt-text = "Smartphones with various Blixt wallet UI screens"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Blixt wallet
+
+## How do we help users new to lightning start using their wallet?
+
+The [Bitcoin Design Community](https://bitcoindesign.slack.com/join/shared_invite/zt-10sxfovaq-isViijl4RThKRs_TsAQnuA#/shared-invite/email) ran a design sprint on an open-source lightning wallet and during this sprint we collaboratively focused on re-designing one aspect of the user flow, the **onboarding journey**. During this collaborative effort, [we got together](https://www.youtube.com/watch?v=nctibe_a4rM&list=PLpV0KfVOMojYywq1Blp3SGx62cSyoRfHt&index=3) and as a group of designers and researchers we aimed to improve the user experience by using good design principles incorporating as much feedback from users as possible. 
+
+<p class="h3">Blixt Wallet is a non-custodial open-source Bitcoin Lightning Wallet for Android and iOS and aims to serve users who are new to lightning, but are familiar with bitcoin and want to try out the lightning network for the first time.</p>
+
+The current user base of Blixt tends to be more advanced users and Blixt would like to move towards catering more to users who want to try out the lightning network as well. 
```

Comment:
Just a suggestion, if I lost a context with this suggestion, feel free to ignore.
```suggestion
The current user base of Blixt tends to be more advanced and Blixt would like to move towards catering to beginners. 
```

Reply:
Good suggestion